### ISSUE TITLE
Class with only property false positive on empty code block

### ISSUE THREAD

**Issue by mikefechner** [Riverside-Software/sonar-openedge#718]
I have this class here:

```
BLOCK-LEVEL ON ERROR UNDO, THROW.

USING Consultingwerk.SmartFramework.Authentication.* FROM PROPATH .

CLASS Consultingwerk.SmartFramework.Authentication.AuthenticationServices:

    DEFINE PUBLIC PROPERTY UserBusinessEntity AS CHARACTER NO-UNDO
        INITIAL 'Consultingwerk.SmartFramework.Authentication.UserBusinessEntity':U
    GET.
    PRIVATE SET.

END CLASS.
```

And it's alerting the CLASS block as empty without comment:

![image](https://user-images.githubusercontent.com/13220627/63642796-2a2d9c80-c6c5-11e9-8818-5af2ca1b96a7.png)

**Comment by movedoa** [comment]
https://github.com/Riverside-Software/sonar-openedge/issues/669

**Comment by mikefechner** [comment]
Thanks @movedoa 

@gquerret I don't think marking it as FP in SQ is the solution. IMHO there should be an exception to your rule about DEFINE *PUBLIC* PROPERTY - as those may be there for a good reason.

**Comment by gquerret** [comment]
What's your use case for classes with only public properties ?

**Comment by mikefechner** [comment]
They are actually PUBLIC STATIC. It's a common solution for constants. 

And before you ask :-) According to clean-code philosophy, a comment should not be required when it's named self-explaining.

**Comment by movedoa** [comment]
Another usecase for classes like this is just as a collection of values. When you have defined members this is way better than a string list for example.

**Comment by gquerret** [comment]
Fixed in develop

**Comment by mikefechner** [comment]
Merci!
